components/mailbox/src/Mailbox.svelte

```diff
@@ -246,6 +246,13 @@
           currentPage,
         );
       }
+      if (currentlySelectedThread) {
+        currentlySelectedThread.messages =
```

Comment:
This can be simplified to just
```
currentlySelectedThread = {
   ...currentlySelectedThread,
   messages: currentlySelectedThread.messages?.map((currentMessage) =>
     currentMessage.id === message.id ? message : currentMessage,
   ) ?? [],
}
```

Reply:
Modified! Thanks